frontend/test/specs/event/eventSpec.js

```diff
@@ -54,4 +55,33 @@
             expect(event.end_time).toEqual(end_time_conv);
         });
     });
+
+    describe('addFollower()', () => {
```

Comment:
add before each for this: `const event = new Event(eventData, institution_key); expect(event.followers).toEqual([]);`

Reply:
but there is only one test...